woocommerce/woocommerce-android
# Shipping lines in order details

Closes: #11394 

### Why
As part of Extended Shipping Lines Support, we want to add support for multiple shipping lines.

### Description
This PR supports displaying multiple shipping lines in the order details screen. It creates a new shipping section and relies on the cached shipping methods to display the shipping details.


### Testing instructions
1. Open an order with shipping lines
2. Check that all sipping lines are displayed in the shipping section

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/bede889c-ca92-4d89-bb29-bab1aa7b0ac4
